### DISCUSSION TITLE
How to connect to NocoDB service from local network?

### DISCUSSION THREAD

**Discussion by woodroom** [nocodb/nocodb#759] [Discord Threads]
Hello.
In the video examples of starting the NocoDB service, the address used is localhost:8080/dashboard/#/
Is it possible to access the service from the local network?  Is it enough to change localhost to IP address for this?
If such access is possible and such access is possible, and it is not enough just to change the address in the link, please add to the documentation how to do it.

**Comment by wingkwong** [comment]
Extracted from [here](https://github.com/nocodb/nocodb#environment-variables)

Variable | Mandatory | Comments | If absent
-- | -- | -- | --
NC_BACKEND_URL | No | Custom Backend URL | ``http://localhost:8080`` will be used

The backend package also takes the frontend package. You may try specify ``NC_BACKEND_URL``.
